sig-architecture/api-review-process.md

```diff
@@ -4,8 +4,8 @@
 
 # Process Overview and Motivations
 
-To preserve usability and consistency in Kubernetes APIs, changes and additions require oversight.
-The API review process is intended to maintain logical and functional integrity of the API over time,
+To preserve usability and consistency in Kubernetes core APIs, changes and additions require oversight.
```

Comment:
I would like to get the idea into this doc somehow that "core api" == "has *.k8s.io API Group". That would make it very clear, what do you think?

Reply:
> I would like to get the idea into this doc somehow that "core api" == "has *.k8s.io API Group". That would make it very clear, what do you think?

If we agree that k8s.io is protected as described in the original doc, we can probably avoid trying to declare a special name for it.  I'm in favor of a new domain for sig APIs that are not api-reviewed.

Reply:
Addressed below.